tests/functional/api/test_query.py

```diff
@@ -3,7 +3,7 @@
 
 def test_basic_query(eth_tester_provider):
     chain.mine(3)
```

Comment:
Note that latent failure that there is actually 4 blocks, the genesis block and the 3 mined blocks